### DISCUSSION TITLE
iOS的xlist具体使用教程

### DISCUSSION THREAD

**Discussion by haihao0110** [AlistGo/docs#290] [Q&A]
翻遍了alist文档   就是没有找到苹果端iOS的Xlist的使用教程，作者大大能不能发个详细的教程啊。iOS端的不会部署啊，都是电脑端的教程

**Comment by BlueSkyXN** [comment]
如果你问出这个问题了，那么你就别指望学会在iOS使用了，其他东西你搞不定
请用远程部署
